Python/specialize.c

```diff
@@ -2560,14 +2560,35 @@ static int
 binary_op_extended_specialization(PyObject *lhs, PyObject *rhs, int oparg,
                                   _PyBinaryOpSpecializationDescr **descr)
 {
-    size_t n = sizeof(binaryop_extend_descrs)/sizeof(_PyBinaryOpSpecializationDescr);
-    for (size_t i = 0; i < n; i++) {
-        _PyBinaryOpSpecializationDescr *d = &binaryop_extend_descrs[i];
+    typedef _PyBinaryOpSpecializationDescr descr_type;
+    size_t size = sizeof(binaryop_extend_builtins)/sizeof(descr_type);
```

Comment:
Don't we have some macro for this computation? Something like Py_ARRAY_SIZE?

Reply:
Not aware of one.

Reply:
Ah it was `Py_ARRAY_LENGTH` not `Py_ARRAY_SIZE`